src/checkers/inference/solver/util/Constants.java

```diff
@@ -0,0 +1,26 @@
+package checkers.inference.solver.util;
+
+/**
+ * String constants.
+ * 
+ * @author jianchu
+ *
+ */
+public class Constants {
```

Comment:
This `Constants` class contains a mixed set of different concepts --- 1) for Configuration constants 2) for backEnd classpath string 3) for TURE/FALSE literals 4) for some slot names...

Maybe separate them would be better? (As they are belong to different categories)..

One obvious benefit would be have finer control when applying those constants in the code (e.g. I can manage BackEndType by `BackEndType` enum class, and have some `backEndNameToEnumValue()` and `enumValueToClassPath()` method in that enum class, then in the `GeneralSolver`, we can have some general code check/validate/load backEnd by using this enum class, then adding a new backEnd would be easy --- just add a new EnumValue and have proper config in that enum class. The same as configuration constants).